local-tests/chainrunner/node.py

```diff
@@ -46,7 +52,23 @@ def stop(self):
     def purge(self):
         """Purge chain (delete the database of the node)."""
         cmd = [self.binary, 'purge-chain', '-y'] + self._stdargs()
-        subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
```

Comment:
why not `check`?

Reply:
Someone was complaing that `purge` on an empty chain throws exception due to check and that makes it impossible to mimic the run_nodes workflow:
1. purge
2. bootstrap
3. run